pkgs/tools/backup/btrbk/default.nix

```
@@ -10,9 +22,9 @@ stdenv.mkDerivation rec {
     sha256 = "153inyvvnl17hq1w3nsa783havznaykdam2yrj775bmi2wg6fvwn";
   };
 
-  nativeBuildInputs = [ asciidoc asciidoctor makeWrapper ];
+  nativeBuildInputs = [ buildPackages.asciidoc buildPackages.asciidoctor makeWrapper ];
 
-  buildInputs = with perlPackages; [ perl DateCalc ];
+  buildInputs = [ bash ] ++ (with perlPackages; [ perl DateCalc ]);
```

```suggestion
  buildInputs = (with perlPackages; [ perl DateCalc ]);
```

I don't think this is needed because we do not compile against it.

---

Ping